README.md

```diff
@@ -5,6 +5,36 @@
 
 This repo contains ansible and CI related assets used for the OCP 3 to 4 migration project, the primary use case for these tools is to integrate with Jenkins and allow the creation of all necessary CI workflows. It will consist of several ansible playbooks, which will prepare a environment for migration purposes. This involves provisioning customer-like cluster deployments with [OA installer](https://github.com/openshift/openshift-ansible), and setup of the migration tools from [mig-controller](https://github.com/fusor/mig-controller).
 
+## Migration controller e2e tests
+
+Viz. [parallel-base](https://github.com/fusor/mig-ci/blob/master/pipeline/parallel-base.groovy) pipeline. The execution is triggered by update in mig-controller repo by parametrized job. Parallel base consists of several parameters to choose the type of the deployment (Agnosticd, OA, or origin3-dev).
+
+Other parameters you could specify:
+
+- `AWS_REGION` - region for clusters to deploy.
+
+- `NODE_COUNT` - number of compute nodes to create. Will be the same for source and target cluster.
+
+- `MASTER_COUNT` - number of master nodes to create. Will be the same for source and target cluster.
+
+- `CLUSTER_NAME` - name of the cluster to deploy. The real deployment name will be created by following convention: `${CLUSTER_NAME}-v3-${BUILD_NUMBER}-${OCP3_VERSION}`. In AWS you can find this value in instance tags under `GUID` label.
+
+- `EC2_KEY` - name of ssh public and private key, which will be imported in OCP3 cluster and allow ssh access. By default is `ci`, but for manual run is recomended `libra`.
+
+- `DEPLOYMENT_TYPE` - type of the ocp3 deployment. Could be `agnosticd`, `OA` or `cluster_up`.
+
+- `MIG_CONTROLLER_REPO` - source repository for mig-controller to test.
+
+- `MIG_CONTROLLER_BRANCH` - source branch for mig-controller to test.
+
+### Cleanup
+
+There is an additional boolean parametr, which will disable workspace cleanup after run - `EC2_TERMINATE_INSTANCES`. In case you want to debug the migration after the run:
+
+1) `ssh root@jenkins-me.v2v.bos.redhat.com`
```

Comment:
GH is public, I would remove any references to internal RH servers , just use ssh Jenkins...